zebra/zebra_rib.c

```diff
@@ -316,12 +316,19 @@ static ssize_t printfrr_zebra_node(struct fbuf *buf, struct printfrr_eargs *ea,
 }
 
 #define rnode_debug(node, vrf_id, msg, ...)                                    \
-	zlog_debug("%s: (%u:%pZNt):%pZN: " msg, __func__, vrf_id, node, node,  \
-		   ##__VA_ARGS__)
+	do {                                                                   \
```

Comment:
These VRF-logging related are like here https://github.com/FRRouting/frr/pull/16798, right? Should we drop from here or close that PR?

Reply:
the changes I have are on top of this patch and I didn't want to disentangle.  Let's get that other one in and then this one can go in.